hull3/marker_functions.sqf

```diff
@@ -112,60 +112,66 @@ hull3_marker_fnc_addUnitMarker = {
 };
 
 hull3_marker_fnc_updateAllMarkers = {
-    if (count hull3_marker_updatableMarkers > 0) then {
-        waitUntil {
+    [
+        {
+            params ["_args", "_id"];
+
+            if (count hull3_marker_updatableMarkers < 1) exitWith {
```

Comment:
The original behavior was to not do anything if there are no markers. Here you remove the handler when there are no more markers, so if there are markers added later, they won't get updated.